ipatests/test_integration/test_ntp_options.py

```diff
@@ -388,3 +388,8 @@ def teardown_method(self, method):
         tasks.uninstall_client(self.client)
         tasks.uninstall_master(self.replica)
         tasks.uninstall_master(self.master)
+
+    @classmethod
```

Comment:
There is nothing special in this `teardown_method` and it is not a part of test suite, so maybe it is better to remove it and let the base `uninstall` do the job.

Reply:
This method is called after each test inside the class and must be kept (the uninstall base method is called only at the end of the class, not at the end of each test).

Reply:
You are right, I should me more attentive.